torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -176,20 +176,31 @@ void TensorPipeAgent::pipeRead(
     std::function<void(const tensorpipe::Error&, Message&&)> fn) {
   pipe->readDescriptor([fn{std::move(fn)}, pipe](
                            const tensorpipe::Error& error,
-                           tensorpipe::Message&& tpMessage) mutable {
+                           tensorpipe::Message tpMessage) mutable {
```

Comment:
Any reason for dropping the rvalue reference type requirement here?

Reply:
The type expected by the `readDescriptor` function takes the Message by value rather than by rvalue-ref, so I found it weird to use a different type here. The two are compatible, so it makes no real difference. In fact, the Message type is movable-only, so...

If the style guide in PyTorch is to specify the rvalue ref in arguments that are expected to be moved in I can change this back.

Reply:
I see. All good, no need to change.